pkg/commands/git_commands/commit_loader.go

```diff
@@ -364,23 +358,42 @@ func (self *CommitLoader) setCommitMergedStatuses(refName string, commits []*mod
 			commits[i].Status = models.StatusMerged
 		}
 	}
-	return commits, nil
+	return commits
 }
 
-func (self *CommitLoader) getMergeBase(refName string) (string, error) {
-	info, err := self.getCurrentBranchInfo()
-	if err != nil {
-		return "", err
+func (self *CommitLoader) getMergeBase(refName string) string {
+	if self.quotedBaseBranches == nil {
+		self.quotedBaseBranches = self.getExistingBaseBranches()
 	}
 
-	baseBranch := "master"
-	if strings.HasPrefix(info.RefName, "feature/") {
-		baseBranch = "develop"
+	if *self.quotedBaseBranches == "" {
+		return ""
 	}
 
-	// swallowing error because it's not a big deal; probably because there are no commits yet
```

Comment:
I think this comment wasn't accurate; there can't be an error when there are no commits yet, because we don't call this function in that case. I think the only reason why this could fail (except for i/o errors or data corruption) is that the base branch doesn't exist. I now handle this error further down by throwing away the cache.